schemas/app/inventory/StockTransfer.json

```diff
@@ -65,6 +65,25 @@
       "abstract": true,
       "fieldname": "backReference",
       "section": "References"
+    },
+    {
+      "fieldname": "isReturn",
```

Comment:
This is kind of a redundant field. If `returnAgainst` already implies that it is a return doc `isReturn` is not needed.

And even if it's added, it can be hidden.

In the `StockTransfer` you can have this:

```javascript
get isReturn(): boolean {
    return !!this.returnAgainst;
}
```

But yeah else remove this field.

Reply:
corrected